src/config.ts

```diff
@@ -138,18 +138,19 @@ function encodeStoredItem(data) {
  * @param {*} data 
  */
 function decodeStoredItem(id: string, data) {
-    if(typeof data !== "string") return data;
-    
-    try {
-        let str = JSON.parse(data);
-        
-        if(!Array.isArray(str)) return data;
-        return new SBMap(id, str);
-    } catch(e) {
-
-        // If all else fails, return the data
-        return data;
+    if(!Config.localConfig[id]) return data;
+
+    if(Config.localConfig[id] instanceof SBMap) {
```

Comment:
Won't it be an array?

Reply:
Config.localConfig[id] should be the config value i think

Reply:
Yea, but chrome stores SBMap as an array right (with decode)

Reply:
Config.localConfig[id] is its old value

Reply:
This is decoding it. Did you mean to use `Config.defaults`?

Reply:
well it should be added when addDefaults() gets run.
however Config.defaults would probably be better to use

Reply:
Is this resolved?